lib/Arch/Arch.cpp

```diff
@@ -863,7 +863,7 @@ DefaultContextAndLifter::DecodeInstruction(uint64_t address,
   inst.SetLifter(std::make_unique<remill::InstructionLifter>(
       this, this->GetInstrinsicTable()));
   if (this->ArchDecodeInstruction(address, instr_bytes, inst)) {
-    return [](uint64_t x) -> DecodingContext { return DecodingContext(); };
+    return [](uint64_t _) -> DecodingContext { return DecodingContext(); };
```

Comment:
Just `[](uint64_t) -> DecodingContext { ... }`.
No need for `_`. (In other changes as well)

As a sidenote, `-> DecodingContext` can also be dropped, I think compiler should be able to deduce this, but I can understand if you want to keep it as part of docs.